Mathlib/Data/Finsupp/Defs.lean

```diff
@@ -673,6 +678,22 @@ theorem erase_zero (a : α) : erase a (0 : α →₀ M) = 0 := by
   classical rw [← support_eq_empty, support_erase, support_zero, erase_empty]
 #align finsupp.erase_zero Finsupp.erase_zero
 
+theorem update_eq_erase (f : α →₀ M) (a : α) : update f a 0 = f.erase a :=
```

Comment:
wouldn't the name `update_zero_eq_erase` be more explicit?

Reply:
Or perhaps `erase_eq_update(_zero)` after flipping the direction?

Reply:
Yes. For a surprising reason, the absence of zero doesn't disturb me in the other direction.

Reply:
I've gone for reversing it then